src/Adapters/Storage/Mongo/MongoStorageAdapter.js

```diff
@@ -692,26 +692,28 @@ export class MongoStorageAdapter implements StorageAdapter {
     fieldNames: string[],
     indexName: ?string,
     caseInsensitive: boolean = false,
-    indexType: any = 1
```

Comment:
Does this have the potential to be a breaking change?

Reply:
Unlikely. I refactored calls that use the `indexType` parameter. All tests pass and if we are missing a test case, then any break would reveal that for the better.